investpy/__init__.py

```diff
@@ -1085,7 +1163,8 @@ def get_fund_recent_data(fund, country, as_json=False, order='ascending', debug=
     id_ = funds.loc[(funds['name'].str.lower() == fund).idxmax(), 'id']
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.